cmd/kusk/cmd/deploy.go

```diff
@@ -107,63 +108,68 @@ var deployCmd = &cobra.Command{
 				if err := k8sclient.Update(ctx, api, &client.UpdateOptions{}); err != nil {
 					return err
 				}
+				fmt.Printf("api.gateway.kusk.io/%s updated\n", api.Name)
 			} else {
 				return err
 			}
 		} else {
 			fmt.Printf("api.gateway.kusk.io/%s created\n", api.Name)
 		}
 
-		if watch {
-			var watcher *filewatcher.FileWatcher
-			absoluteApiSpecPath := file
-			if err != nil {
-				return err
-			}
+		if _, e := url.ParseRequestURI(file); e != nil {
+			if watch {
+				var watcher *filewatcher.FileWatcher
+				absoluteApiSpecPath := file
+				if err != nil {
```

Comment:
What is this `err` check for? I notice there is one beneath it:

```go
				watcher, err = filewatcher.New(absoluteApiSpecPath)
				if err != nil {
					return err
				}
```

Reply:
if watcher.New  fails

Reply:
the error check is pointless here